dist/Tie-File/lib/Tie/File.pm

```diff
@@ -2128,6 +2131,16 @@ then the array C<@gifts> would appear to contain C<("Gold\n",
 "Frankincense\n", "Myrrh\n")>, or (on Win32 systems) C<("Gold\r\n",
 "Frankincense\r\n", "Myrrh\r\n")>.
 
+=head2 C<binmode>
+
+Specify the binmode to set for files opened by Tie::File.
+
+  # make sure Perl is expecting unicode files and strings
+  use utf8;
+  use feature 'unicode_strings';
+  # tie a file with utf8 binmode
+  tie my @array, 'Tie::File', 'array.file', 'binmode' => ':utf8';
```

Comment:
Please suggest ':encoding(UTF-8)' rather than ':utf8' particularly for reading files as the :utf8 layer can result in corrupted strings.

```suggestion
  tie my @array, 'Tie::File', 'array.file', 'binmode' => ':encoding(UTF-8)';
```